app/assets/javascripts/utils/course_date_utils.js

```diff
@@ -21,59 +32,53 @@ const CourseDateUtils = {
       timeFormat = ' HH:mm';
       try {
         timeZoneAbbr = ' ';
-        timeZoneAbbr += moment(datetime).toDate().toString().split('(')[1].slice(0, -1);
+        timeZoneAbbr += datetime.toString().split('(')[1].slice(0, -1);
       } catch (err) {
         timeZoneAbbr = '';
       }
     }
-    const format = `YYYY-MM-DD${timeFormat}`;
-    return moment(datetime).format(format) + timeZoneAbbr;
+    const dateFormat = `yyyy-MM-dd${timeFormat}`;
+    return format(datetime, dateFormat) + timeZoneAbbr;
   },
 
   // Returns an object of minDate and maxDate props for each date field of a course
   dateProps(course) {
-    const startDate = moment(course.start, 'YYYY-MM-DD');
-
+    const startDate = parseISO(course.start);
     const props = {
       end: {
         minDate: startDate
       },
       timeline_start: {
         minDate: startDate,
-        maxDate: moment(course.timeline_end, 'YYYY-MM-DD')
+        maxDate: parseISO(course.timeline_end)
       },
       timeline_end: {
-        minDate: moment(course.timeline_start, 'YYYY-MM-DD'),
-        maxDate: moment(course.end, 'YYYY-MM-DD')
+        minDate: parseISO(course.timeline_start),
+        maxDate: parseISO(course.end)
```

Comment:
Does `parseISO(datetime)` do the same thing as moment(datetime, 'YYYY-MM-DD'), or is this changing the format of these `dateProps` values?

Reply:
The format is not being changed - `moment(datetime, 'YYYY-MM-DD')` simply parsed the given string(`datetime`) according to the given format(`YYYY-MM-DD`) to return a moment date object. 

`parseISO` will fail if the date can't be parsed according to the given format.